ClassNotFoundException: git4idea.repo.GitRepository

Happens with Todosaurus 1.3.0 in IDEA 2025.1 EAP:
```
java.lang.NoClassDefFoundError: git4idea/repo/GitRepository
	at me.fornever.todosaurus.core.git.GitHostingRemoteProvider.provideAll(GitHostingRemoteProvider.kt:27)
	at me.fornever.todosaurus.core.git.ui.wizard.ChooseGitHostingRemoteStep.updateHostingRemotePicker(ChooseGitHostingRemoteStep.kt:106)
	at me.fornever.todosaurus.core.git.ui.wizard.ChooseGitHostingRemoteStep._init(ChooseGitHostingRemoteStep.kt:36)
	at com.intellij.ide.wizard.AbstractWizard.updateStep(AbstractWizard.java:465)
	at me.fornever.todosaurus.core.ui.wizard.TodosaurusWizard.updateStep(TodosaurusWizard.kt:193)
	at com.intellij.ide.wizard.AbstractWizard.updateStep(AbstractWizard.java:393)
	at me.fornever.todosaurus.core.ui.wizard.TodosaurusWizard.doNextAction(TodosaurusWizard.kt:146)
	at com.intellij.ide.wizard.AbstractWizard.proceedToNextStep(AbstractWizard.java:229)
	at me.fornever.todosaurus.core.ui.wizard.TodosaurusWizard.createSouthPanel$lambda$11(TodosaurusWizard.kt:321)
	at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1972)
	at java.desktop/javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2314)
	at java.desktop/javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:407)
	at java.desktop/javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:262)
	at java.desktop/javax.swing.plaf.basic.BasicButtonListener.mouseReleased(BasicButtonListener.java:279)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6676)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3394)
	at java.desktop/java.awt.Component.processEvent(Component.java:6441)
	at java.desktop/java.awt.Container.processEvent(Container.java:2266)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5046)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4871)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4963)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4577)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4518)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2310)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2810)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4871)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:783)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:728)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:98)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:755)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:753)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:752)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:584)
	at com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.kt:529)
	at com.intellij.ide.IdeEventQueue._dispatchEvent$lambda$14(IdeEventQueue.kt:472)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction$lambda$5(AnyThreadWriteThreadingSupport.kt:253)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWithTemporaryThreadLocal(AnyThreadWriteThreadingSupport.kt:202)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:253)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:472)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$12$lambda$11$lambda$10$lambda$9(IdeEventQueue.kt:308)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:858)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$12$lambda$11$lambda$10(IdeEventQueue.kt:307)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$2$lambda$1(IdeEventQueue.kt:952)
	at com.intellij.openapi.application.WriteIntentReadAction.lambda$run$0(WriteIntentReadAction.java:24)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction$lambda$5(AnyThreadWriteThreadingSupport.kt:253)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWithTemporaryThreadLocal(AnyThreadWriteThreadingSupport.kt:202)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:253)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteIntentReadAction(ApplicationImpl.java:928)
	at com.intellij.openapi.application.WriteIntentReadAction.compute(WriteIntentReadAction.java:55)
	at com.intellij.openapi.application.WriteIntentReadAction.run(WriteIntentReadAction.java:23)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$2(IdeEventQueue.kt:949)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$3(IdeEventQueue.kt:963)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:117)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:963)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$12(IdeEventQueue.kt:302)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:342)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
Caused by: java.lang.ClassNotFoundException: git4idea.repo.GitRepository PluginClassLoader(plugin=PluginDescriptor(name=Todosaurus, id=me.fornever.todosaurus, moduleName=me.fornever.todosaurus.core, descriptorPath=me.fornever.todosaurus.core.xml, path=~\AppData\Local\Programs\idea-eap-plugins\Todosaurus, version=1.3.0, package=me.fornever.todosaurus.core, isBundled=false), packagePrefix=me.fornever.todosaurus.core., state=active, parents=PluginDescriptor(name=IDEA CORE, id=com.intellij, moduleName=intellij.platform.vcs.impl, descriptorPath=intellij.platform.vcs.impl.xml, path=~\AppData\Local\Programs\IntelliJ IDEA Ultimate 2\lib, version=251.20015.29, package=null, isBundled=true), PluginDescriptor(name=IDEA CORE, id=com.intellij, moduleName=intellij.platform.vcs.log.impl, descriptorPath=intellij.platform.vcs.log.impl.xml, path=~\AppData\Local\Programs\IntelliJ IDEA Ultimate 2\lib, version=251.20015.29, package=null, isBundled=true), PluginDescriptor(name=IDEA CORE, id=com.intellij, moduleName=intellij.platform.vcs.dvcs.impl, descriptorPath=intellij.platform.vcs.dvcs.impl.xml, path=~\AppData\Local\Programs\IntelliJ IDEA Ultimate 2\lib, version=251.20015.29, package=null, isBundled=true), PluginDescriptor(name=IDEA CORE, id=com.intellij, moduleName=intellij.platform.collaborationTools, descriptorPath=intellij.platform.collaborationTools.xml, path=~\AppData\Local\Programs\IntelliJ IDEA Ultimate 2\lib, version=251.20015.29, package=null, isBundled=true), )
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.kt:157)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	... 68 more
```

The same doesn't break in IDEA 243.

--- a/core/src/main/resources/me.fornever.todosaurus.core.xml
+++ b/core/src/main/resources/me.fornever.todosaurus.core.xml
@@ -7,6 +7,7 @@
 <idea-plugin package="me.fornever.todosaurus.core">
 
     <dependencies>
+        <plugin id="Git4Idea"/>
         <plugin id="com.intellij.tasks"/>
     </dependencies>
 

--- a/gitLab/src/main/resources/me.fornever.todosaurus.gitLab.xml
+++ b/gitLab/src/main/resources/me.fornever.todosaurus.gitLab.xml
@@ -7,6 +7,7 @@
 <idea-plugin package="me.fornever.todosaurus.gitLab">
     <dependencies>
         <module name="me.fornever.todosaurus.core"/>
+        <plugin id="com.intellij.tasks"/>
         <plugin id="org.jetbrains.plugins.gitlab"/>
     </dependencies>
 

